lighthouse-core/audits/third-party-facades.js

```diff
@@ -0,0 +1,220 @@
+ *
+ * Entity: Set of domains which are used by a company or product area to deliver third-party resources
+ * Product: Specific piece of software belonging to an entity. Entities can have multiple products.
+ * Facade: Placeholder for a product which looks likes the actual product and replaces itself with that product when the user needs it.
+ */
+
+/** @typedef {import("third-party-web").IEntity} ThirdPartyEntity */
+/** @typedef {import("third-party-web").IProduct} ThirdPartyProduct*/
+/** @typedef {import("third-party-web").IFacade} ThirdPartyFacade*/
+
+/** @typedef {{product: ThirdPartyProduct, entity: ThirdPartyEntity}} FacadableProduct */
+
+const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+const thirdPartyWeb = require('../lib/third-party-web.js');
+const NetworkRecords = require('../computed/network-records.js');
+const MainResource = require('../computed/main-resource.js');
+const MainThreadTasks = require('../computed/main-thread-tasks.js');
+const ThirdPartySummary = require('./third-party-summary.js');
+
+const UIStrings = {
+  /** Title of a diagnostic audit that provides details about the third-party code on a web page that can be lazy loaded with a facade alternative. This descriptive title is shown to users when no resources have facade alternatives available. A facade is a lightweight component which looks like the desired resource. Lazy loading means resources are deferred until they are needed. Third-party code refers to resources that are not within the control of the site owner. */
+  title: 'Lazy load third-party resources with facades',
+  /** Title of a diagnostic audit that provides details about the third-party code on a web page that can be lazy loaded with a facade alternative. This descriptive title is shown to users when one or more third-party resources have available facade alternatives. A facade is a lightweight component which looks like the desired resource. Lazy loading means resources are deferred until they are needed. Third-party code refers to resources that are not within the control of the site owner. */
+  failureTitle: 'Some third-party resources can be lazy loaded with a facade',
+  /** Description of a Lighthouse audit that identifies the third-party code on the page that can be lazy loaded with a facade alternative. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. A facade is a lightweight component which looks like the desired resource. Lazy loading means resources are deferred until they are needed. Third-party code refers to resources that are not within the control of the site owner. */
+  description: 'Some third-party embeds can be lazy loaded. ' +
+    'Consider replacing them with a facade until they are required. [Learn more](https://web.dev/efficiently-load-third-party-javascript/).',
```

Comment:
We're going to wait for docs before landing, right?

Reply:
Correct

Reply:
```suggestion
    'Consider replacing them with a facade until they are required. [Learn more](https://web.dev/third-party-facades/).',
```